install.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+trap "exit" INT
+
+DIR="spigot/spigotJars/install"
+if [ -d "$DIR" ]; then
+	echo "$DIR already created" > /dev/null
+else
+	mkdir -p $DIR
+fi
+cd $DIR
+
+java8=unknow
+java16=unknow
+java17=unknow
+
+if [[ $java8 == *"unknow"* || $java16 == *"unknow"* || $java17 == *"unknow"* ]]; then # if at least one JAVA is NOT set
+	export IFS=";"
+	for javaVersionPath in $JAVA_HOME; do
+	    if [[ $javaVersionPath == *"1.8"* ]]; then # check if has "1.8"
+	    	java8="$javaVersionPath\\bin\\java.exe"
+	    elif [[ $javaVersionPath == *"16."* ]]; then # check if has "16."
+	    	java16="$javaVersionPath\\java.exe"
+	    elif [[ $javaVersionPath == *"17."* ]]; then # check if has "17."
+	    	java17="$javaVersionPath\\java.exe"
+	    fi
+	done
+
+	if [[ $java8 == *"unknow"* ]]; then
+		if [ -d "./jdk-8" ]; then
+			echo "Failed to find Java 8 and already downloaded. Use unzipped version ..."
```

Comment:
```suggestion
			echo "Java 8 and already downloaded, using this version..."
```

Modifier les autres aussi.